app/work/page.tsx

```diff
@@ -1,8 +1,188 @@
+              <ul>
+                {project.stack.map((item, index) => {
+                  return(
+                    <li key = {index} className="text-xl text-accent">
+                      {item.name}
+                      {index !== project.stack.length -1 && ","}
+                    </li>
+                  )
+                })}
+              </ul>
+              <div className="border border-white/20"></div>
+              <div className="flex items-center gap-4">
+                <Link href={project.live} target="_blank">
+                  <TooltipProvider delayDuration={100}>
+                    <Tooltip>
+                      <TooltipTrigger className="w-[70px] h-[70px] rounded-full bg-white/5 flex justify-center items-center group capitalize">
+                        <BsArrowUpRight className="text-white text-3xl group-hover:text-accent"/>
+                      </TooltipTrigger>
+                      <TooltipContent>
+                        <p>Live project</p>
+                      </TooltipContent>
+                    </Tooltip>
+                  </TooltipProvider>
+                </Link>
+                <Link href={project.github} target="_blank">
+                  <TooltipProvider delayDuration={100}>
+                    <Tooltip>
+                      <TooltipTrigger className="w-[70px] h-[70px] rounded-full bg-white/5 flex justify-center items-center group">
+                        <BsGithub className="text-white text-3xl group-hover:text-accent"/>
+                      </TooltipTrigger>
+                      <TooltipContent>
+                        <p>Github repository</p>
+                      </TooltipContent>
+                    </Tooltip>
+                  </TooltipProvider>
+                </Link>
+              </div>
+            </div>
+          </div>
+          <div className="w-full xl:w-[50%]">
+            <Swiper
+              spaceBetween={30} 
+              slidesPerView={1}
+              className="xl:h-[520px] mb-12"
+              onSlideChange={handleSlideChange}
+            >
+              {
+                projects.map((project, index) => {
+                  return(
+                    <SwiperSlide key={index} className="w-full">
+                      <div className="h-[460px] relative group flex justify-center item-center bg-pink-50/20">
+                        <div className="absolute top-0 bottom-0 w-full h-full bg-black/10 z-10"></div>
+                        <div className="relative w-full h-full">
+                          <Image 
+                            src={project.image}
+                            fill
+                            className="object-cover"
+                            alt=""
+                          />
+                        </div>
+                      </div>
+                    </SwiperSlide>
+                  )
+                })
+              }
+              <WorkSliderBtn
```

Comment:
is this reused anywhere? Or is there a plan for this to be reused somewhere?

Reply:
I thought it might be good to add this as a separate component. Does it have any effect on the performance? @emkkur

Reply:
Not on performance but just the effort to make it. If it's not reused then it's just extra lines of code and effort